src/ServerRequest.php

```diff
@@ -167,6 +167,11 @@ public static function fromGlobals(): ServerRequestInterface
     {
         $method = $_SERVER['REQUEST_METHOD'] ?? 'GET';
         $headers = getallheaders();
+        $disallowEmpty = ['content-type', 'content-length'];
```

Comment:
Should we disallow empty for all headers?

Reply:
I was shared ti do that since some headers are perhaps even allowed empty. We could verify, but I was rather scared to so since it might have unforseen consequences.

I'll browse through the headers supported and check.

Update: some are allowed empty it seems so a we could add to the whitelist those whom cannot.